bitnami/etcd/values.yaml

```
@@ -645,6 +645,9 @@ metrics:
     ##       name: existingSecretName
     ##
     tlsConfig: {}
+    ##
+    ## @param metrics.podMonitor.relabelings [array] Prometheus relabeling rules
```

Sorry, just a cosmetic change I missed before:
```suggestion
    ## @param metrics.podMonitor.relabelings [array] Prometheus relabeling rules
    ##
```
Everything else LGTM!